tests/sentry/integrations/vercel/test_webhook.py

```diff
@@ -274,12 +281,39 @@ def test_manual_vercel_deploy(self):
         ).hexdigest()
 
         response = self.client.post(
-            path=webhook_url,
+            path=self.webhook_url,
             data=DEPLOYMENT_WEBHOOK_NO_COMMITS,
             content_type="application/json",
-            HTTP_X_ZEIT_SIGNATURE=local_signature,
+            **{f"HTTP_X_{self.header}_SIGNATURE": local_signature},
         )
 
         assert response.status_code == 404
         assert "No commit found" == response.data["detail"]
         assert len(responses.calls) == 0
+
+
+class VercelReleasesNewTest(VercelReleasesTest):
+    webhook_url = "/extensions/vercel/delete/"
+    header = "VERCEL"
+
+    @responses.activate
+    def test_release_already_created(self):
+        responses.add(
+            responses.POST,
+            absolute_uri("/api/0/organizations/%s/releases/" % self.organization.slug),
+            json={},
+        )
+
+        self.create_release(
+            project=self.project, version="7488658dfcf24d9b735e015992b316e2a8340d9d"
+        )
+
+        with override_options({"vercel.client-secret": SECRET}):
+            response = self.client.post(
+                path=self.webhook_url,
+                data=EXAMPLE_DEPLOYMENT_WEBHOOK_RESPONSE,
+                content_type="application/json",
+                **{f"HTTP_X_{self.header}_SIGNATURE": SIGNATURE},
+            )
+
+            assert response.status_code == 201
```

Comment:
@MeredithAnya Can we copy the other tests and spec that we have for creating a release? Want to make sure we aren't losing test coverage with the endpoint, even if much of the functionality is tested with the legacy endpoint as well. That way we can just rip out the old tests later instead of having to figure out what parts of tests need to be moved over.

Reply:
@scefali it should already be running all the same tests, that should just be an added one. I made the `VercelReleasesNewTest` inherit from the `VercelReleasesTest`, so it should just run the same ones but with the new `webhook_url`

Reply:
@MeredithAnya Neat! I missed that